packages/gatsby-transformer-remark/src/extend-node-type.js

```diff
@@ -254,7 +254,7 @@ module.exports = (
           return getAST(markdownNode).then(ast => {
             const textNodes = []
             visit(ast, `text`, textNode => textNodes.push(textNode.value))
-            return prune(textNodes.join(` `), pruneLength)
+            return prune(textNodes.join(` `), pruneLength, '\u2026')
```

Comment:
is it possible to just put the actual ellipsis character "…" in the code vs. this? I dislike looking up unicode characters and most terminals/editors display them just fine these days.